utils/travis/dependencies.sh

```diff
@@ -3,11 +3,14 @@
 
 echo "*****************************************"
 echo "Fetching dependencies"
-echo "Using build system: ${BUILD_SYSTEM}"
+! [ -z ${BUILD_VARIANT} ] && echo "Building ${BUILD_VARIANT}"
+! [ -z ${BUILD_SYSTEM} ] && echo "Using build system: ${BUILD_SYSTEM}"
 echo "*****************************************"
```

Comment:
Better always set `$BUILD_SYSTEM` and `$PLATFORM` in common.sh. Then this can be simplified.